scripts/scil_prepare_topup_command.py

```diff
@@ -99,38 +99,43 @@ def main():
 
     acqparams = create_acqparams(gtab, args.readout, args.encoding_direction)
 
-    if not os.path.exists(args.output_directory):
-        os.makedirs(args.output_directory)
+    if not os.path.exists(args.out_directory):
+        os.makedirs(args.out_directory)
 
-    acqparams_path = os.path.join(args.output_directory, 'acqparams.txt')
+    acqparams_path = os.path.join(args.out_directory, 'acqparams.txt')
     np.savetxt(acqparams_path, acqparams, fmt='%1.4f', delimiter=' ')
 
-    rev_b0_img = nib.load(args.reverse_b0)
-    rev_b0 = rev_b0_img.get_data()
+    rev_b0_img = nib.load(args.in_reverse_b0)
+    rev_b0 = rev_b0_img.get_fdata(dtype=np.float32)
 
-    dwi_image = nib.load(args.input_dwi)
-    dwi = dwi_image.get_data()
+    if len(rev_b0.shape) == 4:
+        logging.warning("Reverse B0 is 4D. To speed up Topup, the mean of all "
+                        "reverse B0 will be taken.")
+        rev_b0 = np.mean(rev_b0, axis=3, dtype=rev_b0.dtype)
```

Comment:
You load it as float32, so it will always be float32. was that intentional?

Reply:
It is because I changed the get_data for get_fdata. Thanks for that